### PR TITLE
don't call Avatar::simulate() on fading avatars

### PR DESCRIPTION
**Pull request by AndrewMeadows**

### CODE REVIEW

**Comment by AndrewMeadows** [comment]
This is a minor optimization: don't call `Avatar::simulate()` on fading avatars, as per internal manuscript case 17446:
https://highfidelity.manuscript.com/f/cases/17446/

**Test Plan**:

**(1)** Login with two users

**(2)** Have UserA watch as UserB leaves the domain.
--> the leaving avatar should appear to freeze animation and "fade out" using some shader effect
(removing this simulate() call doesn't change the appearance of the avatar as it fades)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11953/HighFidelity-Beta-PR13785-9b0dcb4.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11953/HighFidelity-Beta-Interface-PR13785-9b0dcb4.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11953/HighFidelity-Beta-PR13785-9b0dcb4.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11953/HighFidelity-Beta-Interface-PR13785-9b0dcb4.exe

File: interface/src/avatar/AvatarManager.cpp [hunk 1]
```diff
@@ -365,8 +365,6 @@ void AvatarManager::simulateAvatarFades(float deltaTime) {
             }
             avatarItr = _avatarsToFade.erase(avatarItr);
         } else {
-            const bool inView = true; // HACK
-            avatar->simulate(deltaTime, inView);
```
[the remark this answers is not in the document]
Comment (AndrewMeadows) [reply interface/src/avatar/AvatarManager.cpp]: Oh right, at the moment the avatar is being removed from the fading list.  That doesn't happen very often compared to the other case but yeah, I'll remove it.

[the remark this answers is not in the document]
Comment (AndrewMeadows) [reply interface/src/avatar/AvatarManager.cpp]: Huh... actually we don't need any of that code.  There is no need to `simulate()` or `animateScaleChanges()` for fading avatars.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11972/HighFidelity-Beta-PR13785-8153e66.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11972/HighFidelity-Beta-PR13785-8153e66.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11972/HighFidelity-Beta-Interface-PR13785-8153e66.exe

**Comment by qaAdamSmith82** [comment]
Pass @conklin94122 

Master and the PR had same results

**Comment by conklin94122** [comment]
According to the Manuscript ticket this is a 73.0 PR, which can not be QA Approved yet. When we branch 72.0 this can be rebuilt with then current master and retested. If the PM changed this to a 72.0 PR please let me know.

**Comment by samcake** [comment]
This has been qa approved and no more commit since then
and we want it in 72, merging

**Pull request merged**